### ISSUE TITLE
drop spine

### ISSUE THREAD

**Issue by therealglazou** [therealglazou/epub31review#3]
edasfr thinks spine should be dropped, could be recreated from link rel=next/prev in content document

Source: https://n.survol.fr/n/e0-le-retour-du-fils-de-la-vengeance

**Comment by dauwhe** [comment]
I also am concerned about duplication between spine, toc, and rel=prev/next. But I'd prefer we keep some sort of spine, likely as a TOC, as I think it's easier to author and maintain in a single place. Constructing a spine from rel would involve parsing every HTML file in an e0.

**Comment by therealglazou** [comment]
> Constructing a spine from rel would involve parsing every HTML file in an e0

Right. But that might be still necessary to construct a unique view of a multi-chapter book...
